Add spanish translation

warbacon opened the pull request:
This adds a translation into European Spanish. Would it be appropriate to copy this translation for other types of Spanish as well?

warbacon committed:
Add es_es.json

This adds translation into European Spanish.

NotRyken commented:
If the correct translation happens to be the same you could, but otherwise probably not.

warbacon commented:
This is a correct translation for Argentinean, Chilean, Ecuadorian, Mexican, Venezuelan, and Uruguayan Spanish, as European Spanish is the most neutral and easily understood across Latin American countries. While some words might be translated differently depending on the country, the translation I provided remains correct.

If someone wishes to be more inclusive, they can simply modify my translation accordingly.

NotRyken commented:
Based on that description I think it would be appropriate to copy the translation.

warbacon committed:
Add translation for all remaining Spanish types

This includes Argentinean, Chilean, Ecuadorian, Mexican, Venezuelan, and
Uruguayan Spanish.

warbacon commented:
Done :)
